Tests/SwiftProtobufTests/Test_Map_JSON.swift

```diff
@@ -285,6 +285,17 @@ class Test_Map_JSON: XCTestCase, PBTestHelpers {
         XCTAssertEqual(decoded.mapInt32Enum, [1: .foo, 3: .baz])
     }
 
+    func testMapInt32Enum_JSONdecodingOptions() {
+        var options = JSONDecodingOptions()
+        options.ignoreUnknownFields = true
+
+        // Test_Enum covers the single and repeated field cases.
+
+        assertJSONDecodeSucceeds("{\"mapInt32Enum\": {\"1\": \"NEW_VALUE\"}}", options: options) { (m: MessageTestType) -> Bool in
+          m.mapInt32Enum == [:]
+        }
+    }
+
```

Comment:
As above, doe need a test case to verify we reject maps containing unknown string enums without this option?

Reply:
same.